### ISSUE TITLE
Parse out and index CamelCase strings by UpperCase letters

### ISSUE THREAD

**Issue by ManiacDC** [ManiacDC/TypingAid#27]
hotkeyguy said:

Using it for programming, that means many "artificial" words in camel case it would be much easier to find the desired word when any character from left to right would be taken into consideration.
An example: I want to insert ThisIsAnExample, so I must type thi and select from the list. My opinion is that it would be easier when I could write e. g. tia or any other valid sequence of chars. The lists would then be often much shorter.

---

So basically you are saying that I should have a CamelCase flag (at the word level? at the preferences level? I'd have to think about that)... and then add an alternative search term to the index by taking only the uppercase letters. So if I type "This", "ThisIsAnExample" will show up, but if I type "tia", "ThisIsAnExample" would show up yet again. I'd be open to the idea... I've added it to the future features.
